R/render-cached-plot.R

```diff
@@ -146,6 +146,7 @@
 #'   possible pixel dimension. See \code{\link{sizeGrowthRatio}} for more
 #'   information on the default sizing policy.
 #' @param res The resolution of the PNG, in pixels per inch.
+#' @param alt Optional parameter to pass `alt` to the img-tag.
```

Comment:
Shiny isn't configured to use markdown in the documentation, so it should be `\code{alt}` instead of `` `alt` ``.

Also, it should be `\code{img} tag` (or `\code{<img>} tag`).